### PR TITLE
Vaults

### PR DESCRIPTION
**Pull request by Haxxer**

### CODE REVIEW

**Comment by diwako** [comment]
About the cyberpunk red filter changes. Is it possible to run the dev version of this branch to give feedback?

**Comment by Haxxer** [comment]
@diwako I would not recommend it, as this one contains a _TON_ of changes that aren't compatible with the current released version of Item Piles. You would not be able to switch back.

**Comment by diwako** [comment]
> @diwako I would not recommend it, as this one contains a _TON_ of changes that aren't compatible with the current released version of Item Piles. You would not be able to switch back.

That is why testing takes place on the test system and not the prod system ;)

**Comment by Haxxer** [comment]
As long as you're aware of the risks :) I'll put out a dev release soon

**Comment by Haxxer** [comment]
@diwako You can test the latest dev release with this:

https://raw.githubusercontent.com/fantasycalendar/FoundryVTT-ItemPiles/next/module.json

Please do test on a fresh world; if you test on a world with item piles, they will be migrated to this version, and you cannot downgrade.

File: src/systems/cyberpunk-red-core.js [hunk 1]
```diff
@@ -15,11 +15,15 @@ export default {
...
     {
       "path": "type",
-      "filters": "role,skill,criticalInjury"
+      "filters": "role, skill, criticalInjury"
     },
     {
       "path": "system.type",
-      "filters": "cyberwareInternal,cyberwareExternal,fashionware"
+      "filters": "cyberwareInternal, cyberwareExternal, fashionware"
+    },
```
Comment (diwako) [inline src/systems/cyberpunk-red-core.js:23]: Justed tested it on a new world. this filter option is not needed anymore with the additional below. It currently prevents placing a few cyberware into piles as well.

Comment (Haxxer) [reply src/systems/cyberpunk-red-core.js:23]: I'll remove that before release, thanks! It's hard to know what exactly is needed for each system :D

**Comment by diwako** [comment]
Probably known when drag and dropping item from char sheet on brand new world.

Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'actor')
[Detected 1 package: item-piles]
    at PrivateAPI._dropItem (private-api.js:1553:41)
    at PrivateAPI._dropData (private-api.js:1438:19)

**Comment by Haxxer** [comment]
Aye, dang, I missed a few `?` in my code. I'll release another fix.

**Comment by Haxxer** [comment]
Done, it's up

**Comment by diwako** [comment]
> Done, it's up

Works now! Vaults I have not yet looked into.

**Pull request merged**